spyder/plugins/editor/plugin.py

```diff
@@ -2701,6 +2706,35 @@ def re_run_last_cell(self):
         editorstack = self.get_current_editorstack()
         editorstack.re_run_last_cell()
 
+    @Slot(int)
+    def save_bookmark(self, num):
+        """Save current cursor position as bookmark in slot based
+        on number of shortcut e.g Ctrl+Shift+1 for slot 1"""

```

Comment:
Per PEP 257, docstrings should have a one-line summary, following by a blank line then additional details if necessary.